components/badge/badge.css

```diff
@@ -0,0 +1,114 @@
+/* parent selector */
+
+.badge {
+  position: relative;
+}
+
+/* Icon badges */
+
+.icon {
+  font-size: var(--size-12);
+  cursor: pointer;
+}
+
+.unit {
+  position: absolute;
+  border-radius: var(--radius-rounded);
+  min-width: var(--size-12);
+  height: var(--size-12);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: var(--size-3);
+  font-size: var(--size-9);
+  font-weight: var(--weight-medium);
+}
+
+.top-right {
```

Comment:
could go in utilities

Reply:
Done, created new utilities.css, will use it for its intended purpose